src/DashboardUI/src/pages/application/estimation-tool/EstimationToolMap.tsx

```diff
@@ -18,6 +21,62 @@ interface EstimationToolMapProps {
 
 export function EstimationToolMap(props: EstimationToolMapProps) {
   const { state, dispatch } = useConservationApplicationContext();
+  const { setMapBoundSettings } = useMapContext();
+
+  useEffect(
```

Comment:
Eh - probably use arrow function unless you had a specific reason not to use the normal syntax on the `useEffect`

```suggestion
  useEffect(() => {
```

Reply:
updated